deployment/articles/03-installing-a-search-engine/02-elasticsearch/04-configuring-the-connector.markdown

```diff
@@ -0,0 +1,253 @@
+@product@ the IP address or host name where search requests should be sent. If
+using System Settings, set the value in the *Transport Addresses* property.
+
++$$$
+
+**Note:** In an Elasticsearch cluster you can list the transport addresses for multiple Elasticsearch nodes as a comma-separated list in the `transportAddresses` property. If you set only one transport address, @product@ loses contact with Elasticsearch if that node goes down.
+
+$$$
+
+On the Elasticsearch side, set the `network.host` property in your
+`elaticsearch.yml` file. This property simultaneously sets both the *bind host*
+(the host where Elasticsearch listens for requests) and the *publish host*
+(the host name or IP address Elasticsearch uses to communicate with other
+nodes). See
+[here](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-network.html)
+for more information.
+
+## Clustering Elasticsearch in Remote Operation Mode 
+
+To cluster Elasticsearch, first set `node.max_local_storage_nodes` to
+be something greater than `1`. When you run the Elasticsearch start script,
+a new local storage node is added to the cluster. If you want four nodes running
+locally, for example, run `./bin/elasticsearch` four times. See
+[here](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-node.html#max-local-storage-nodes)
+for more information.
+
+Configure the number of shards and replicas in the Elasticsearch 6 connector,
+using the `indexNumberOfShards` and `indexNumberOfReplicas` properties to
+specify the number of primary shards and number of replica shards, respectively.
+Elasticsearch's default configuration works for a cluster of up to ten nodes,
+since the default number of shards is `5` and the default number of replica
+shards is `1`.
+
++$$$
+
+**Note:** Elasticsearch uses the [Zen Discovery Module](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-discovery-zen.html) by default, which provides unicast discovery. Additionally, nodes in the cluster communicate using the [Transport Module](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-transport.html), through TCP. See the Elasticsearch documentation for the available properties (to be set in the `elasticsearch.yml` file), and the @product@ Elasticsearch connector's settings for the connector's available settings.
+<!--reference article](discover/reference/-/knowledge_base/7-1/elasticsearch-settings)-->
+
+At a minimum, provide the list of hosts (as `host:port`)  to act as gossip routers during unicast discovery in the `elasticsearch.yml`:
+
+    discovery.zen.ping.unicast.hosts: ["node1.ip.address", "node2.ip.address"]
+
+For example,
+
+    discovery.zen.ping.unicast.hosts: ["10.10.10.5", "10.10.10,.5:9305"]
+
+For more information on configuring an Elasticsearch cluster, see the
+documentation on 
+[Elasticsearch Index Settings](https://www.elastic.co/guide/en/elasticsearch/guide/6.5/_index_settings.html).
```

Comment:
This the "guide", not the versioned reference docs we use most often. There's no 6.5 link for this.